src/apps/dhedge-v2/optimism/dhedge-v2.staking-v2.token-fetcher.ts

```diff
@@ -0,0 +1,57 @@
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
```

Comment:
This position shouldn't be combined with the app token. It should be it's own position and it should extend `CustomContractPositionTemplatePositionFetcher`